core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20230316_fastpass_grants.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="dummy" author="garwood" id="create_FASTPASS_GRANTS">
+        <createTable tableName="FASTPASS_GRANTS"
+                     remarks="Records information about fastbass grants.">
+            <column name="id" type="BIGINT" autoIncrement="true">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="USER_ID"
+                    type="VARCHAR(254)">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="WORKSPACE_ID"
+                    type="VARCHAR(254)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="RESOURCE_TYPE"
+                    type="VARCHAR(254)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="RESOURCE_NAME"
+                    type="VARCHAR(254)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="ROLE_NAME"
+                    type="VARCHAR(254)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="EXPIRATION"
+                    type="DATETIME"
```

Comment:
Other tables are using `TIMESTAMP(6)`. We probably should as well for consistency's sake.